templates/wechatgame/first-screen.js

```diff
@@ -461,20 +461,22 @@ function start(alpha, antialias, useWebgl2) {
     programProgress = initShaders(VS_PROGRESSBAR, FS_PROGRESSBAR);
     tick();
     return Promise.all([
-        loadBackground('background.png').then(() => {
-          updateBgVertexBuffer();
-          updateBgTexture();
-        }),
-        loadSlogan('slogan.png').then(() => {
-          updateSloganVertexBuffer();
-          updateSloganTexture();
-        }),
+        //logo should be loaded earlier than slogan
         loadImage('logo.png').then(() => {
             updateVertexBuffer();
             updateLogoTexture();
+        }).then(() => {
+            loadSlogan('slogan.png').then(() => {
+                updateSloganVertexBuffer();
+                updateSloganTexture();
+            })
```

Comment:
```suggestion
            return loadSlogan('slogan.png').then(() => {
                updateSloganVertexBuffer();
                updateSloganTexture();
            });
```

need to return the new promise

Reply:
Why loadSlogan twice?

Reply:
fixed.